vmdb/lib/miq_automation_engine/service_models/miq_ae_service_orchestration_template_cfn.rb

```diff
@@ -0,0 +1,3 @@
+module MiqAeMethodService
+  class MiqAeServiceOrchestrationTemplateCfn < MiqAeServiceOrchestrationTemplate; end
```

Comment:
@chessbyte CFN (CloudFormation) and HOT are two template formats. The name CFN or cfn is widely used.